mlos_bench/setup.py

```diff
@@ -24,33 +25,44 @@
     warning(f"setuptools_scm failed to find git version, using version from _version.py: {e}")
 
 
-extra_requires = {
+extra_requires: Dict[str, List[str]] = {
     # Additional tools for extra functionality.
-    'azure': 'azure-storage-file-share',
-    'sqlalchemy': 'sqlalchemy',
+    'azure': ['azure-storage-file-share'],
+    'storage-sql-duckdb': ['sqlalchemy', 'duckdb_engine'],
+    'storage-sql-mysql': ['sqlalchemy', 'mysql-connector-python'],
```

Comment:
Provides aliases for different types of db backends we support.

Reply:
oh nice